databricks/koalas/frame.py

```diff
@@ -1470,6 +1470,84 @@ def to_excel(self, excel_writer, sheet_name="Sheet1", na_rep="", float_format=No
         return validate_arguments_and_invoke_function(
             kdf.to_pandas(), self.to_excel, pd.DataFrame.to_excel, args)
 
+    def to_records(self, index=True, convert_datetime64=None,
+                   column_dtypes=None, index_dtypes=None):
+        """
+        Convert DataFrame to a NumPy record array.
+
+        Index will be included as the first field of the record array if
+        requested.
```

Comment:
also need a note warning users. see to_csv / to_json for examples

Reply:
Thanks for mentioning.
Added.